### PR TITLE
dev-python/pypam: update to EAPI 8, add Python 3.10

### PR DESCRIPTION
**Pull request by hydrapolic**
Signed-off-by: Tomáš Mózes <hydrapolic@gmail.com>

### CODE REVIEW

File: dev-python/pypam/pypam-0.5.0-r7.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	"${FILESDIR}/${P}-stricter.patch"
+)
+
+src_compile() {
+	append-cflags -fno-strict-aliasing
+	distutils-r1_src_compile
+}
+
+python_test() {
+	"${PYTHON}" tests/PamTest.py
```
Comment (thesamesam) [inline dev-python/pypam/pypam-0.5.0-r7.ebuild:44]: If this fails, what's going to happen? 😃

Comment (hydrapolic) [reply dev-python/pypam/pypam-0.5.0-r7.ebuild:44]: Life goes on ;)

Comment (hydrapolic) [reply dev-python/pypam/pypam-0.5.0-r7.ebuild]: Added || die

**Comment by hydrapolic** [comment]
Needed for xen-4.16 in https://github.com/gentoo/gentoo/pull/23442, continuing there.

**Pull request closed without merging**